edx_exams/apps/api/v1/urls.py

```diff
@@ -1,4 +1,13 @@
 """ API v1 URLs. """
+from django.urls import path
+
+from . import views
 
 app_name = 'v1'
-urlpatterns = []
+urlpatterns = [
+    path('lti_start_proctoring', views.lti_start_proctoring),
+    path('oidc_auth_request', views.lti_oidc_authentication),
```

Comment:
if this isn't mandated to be oidc_auth_request in the path it would be clearer to have these all start with lti

or possibly just have them more like a hierarchy all under lti/